common/src/main/java/com/wynntils/features/user/overlays/MobTotemTrackingFeature.java

```diff
@@ -0,0 +1,127 @@
+
+        if (mobTotemTimerOverlay.ticksUntilUpdate <= 0) {
+            mobTotemTimerOverlay.ticksUntilUpdate = MobTotemTimerOverlay.TICKS_PER_UPDATE;
+            mobTotemTimerOverlay.updateRenderTaskCache();
+        }
+    }
+
+    public static class MobTotemTimerOverlay extends Overlay {
+        static final int TICKS_PER_UPDATE = 5;
+
+        @Config
+        public TextShadow textShadow = TextShadow.OUTLINE;
+
+        private TextRenderSetting textRenderSetting;
+
+        private int ticksUntilUpdate = 0;
+        private List<TextRenderTask> renderTaskCache;
+
+        protected MobTotemTimerOverlay() {
+            super(
+                    new OverlayPosition(
+                            330,
+                            -5,
+                            VerticalAlignment.Top,
+                            HorizontalAlignment.Right,
+                            OverlayPosition.AnchorSection.TopRight),
+                    new GuiScaledOverlaySize(120, 35));
+
+            updateTextRenderSetting();
+        }
+
+        @Override
+        public void render(
+                PoseStack poseStack, MultiBufferSource.BufferSource bufferSource, float partialTicks, Window window) {
+            if (renderTaskCache == null) {
+                updateRenderTaskCache();
+            }
+
+            BufferedFontRenderer.getInstance()
+                    .renderTextsWithAlignment(
+                            poseStack,
+                            bufferSource,
+                            this.getRenderX(),
+                            this.getRenderY(),
+                            renderTaskCache,
+                            this.getWidth(),
+                            this.getHeight(),
+                            this.getRenderHorizontalAlignment(),
+                            this.getRenderVerticalAlignment());
+        }
+
+        @Override
+        public void renderPreview(
+                PoseStack poseStack, MultiBufferSource.BufferSource bufferSource, float partialTicks, Window window) {
+            BufferedFontRenderer.getInstance()
+                    .renderTextWithAlignment(
+                            poseStack,
+                            bufferSource,
+                            this.getRenderX(),
+                            this.getRenderY(),
+                            new TextRenderTask("Mob Totem (Player) at [-105, 58, 3948] (4:11)", textRenderSetting),
+                            this.getWidth(),
+                            this.getHeight(),
+                            this.getRenderHorizontalAlignment(),
+                            this.getRenderVerticalAlignment());
+        }
+
+        void updateRenderTaskCache() {
+            renderTaskCache = Models.MobTotem.getMobTotems().stream()
+                    .map(mobTotem -> new TextRenderTask(
+                            "Mob Totem (" + mobTotem.getOwner() + ") at " + mobTotem.getLocation() + " ("
```

Comment:
Just as with the shaman totems, I think the coordinates is not very helpful. I know I introduced them myself with the original mob totem implementation in Legacy :-) but I have been regretting it ever since. I suggest changing this to a distance instead, that is at least possible for the human brain to parse during a grinding session. (Or make it configurable, but I honestly don't know how the coordinates ever is going to be helpful).